rfcs/20241015-result-accuracy.md

```diff
@@ -0,0 +1,102 @@
+# [RFC] Add result accuracy to transcendental unary ops
+
+Status: In Review<br/>
+Initial version: 10/15/2024<br/>
+Last updated: 10/15/2024<br/>
+Discussion thread:
+
+## Overview
+
+This RFC proposes adding a new attribute `result_accuracy` to the following
+transcendental unary ops: `exp`, `expm1`, `log`, `logp1`, `logistic` and `tanh`.
```

Comment:
Is there a reason sqrt and rsqrt is not in the list?

Reply:
Yeah, it should be should be supported for sqrt, rsqrt, cos, sin and recp too. Will update the doc.

Reply:
What is `recp`? I don't think we have an op for that

Reply:
FYI `atan2` and `tan` are the two remaining transcendental ops not in this list.

Reply:
recp is divide. So the complete list will be sin, cos, tan, divide, sqrt, rqrt, cbrt, divide, exp, expm1, log, logp1, logistic and erf. Is erf supported?

Reply:
This would make divide the only binary op on this list.. which feels strange. Is this right that this proposal should be extended in scope to binary ops?

Reply:
We're able to support accuracy selection for reciprocal, but stablehlo doesn't have a reciprocal op. Is adding a reciprocal op a possible option?

Reply:
I left out divide for now.